src/__tests__/index.test.ts

```diff
@@ -21,6 +21,8 @@ describe('Index', () => {
       'Footer',
       'Form',
       'useFormContext',
+      'FormGroup',
+      'useFormGroupContext',
```

Comment:
Do we want to be exposing the `useFormGroupContext` function? I.e. is there any use for it outside of the library?